### ISSUE TITLE
Define Dropout Value for Cycle & Pix2Pix

### ISSUE THREAD

**Issue by Tonks684** [junyanz/pytorch-CycleGAN-and-pix2pix#1252]
I have managed to turn dropout on for both pix2pix and cycle gan during training and in inference. 

I would now like to explore the impact different values for dropout has on the predictions that are drawn from  of each of the two models when running inference on the same dataset multiple times. How can I determine the value of dropout used? The only parameters I see within options are a binary True or False.

**Comment by aaroncoyner** [comment]
The value for dropout is hard-coded as 0.5 (see models/networks.py: lines 415 and 525). You can try changing them to see what happens.
